doc/faq.rst

```diff
@@ -1,6 +1,131 @@
 Frequently Asked Questions
 ==========================
 
+How do I get started with Relate?
+---------------------------------
+At the start of a course, there are a few steps required to get going.
+We assume that the Relate server is already installed and that you have
+a user account there. Your account will need to have sufficient
+privileges to create a course. (You can tell whether that's the case by
+checking that there is a 'Set up new course' button at the bottom of the main
+Relate page.)
+
+Getting everything set up
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-   Start by creating a Git repository on a hosting site (Github, Gitlab,
+    Bitbucket, or similar) for your course.  Likely you will want this to be a
+    private repository, to prevent students from seeing solutions to your
+    assignments.
+
+-   If you have a course repository from a prior semester, you can start by
+    pushing its content to your new repository.
+
+    If you're starting from scratch, you can use the
+    `sample course <https://github.com/inducer/relate-sample>`__.
+    You can either create your own and use it as a guide, or use
+    it in its entirety and just make modifications.
+
+-   Now you are ready to click that 'Set up new course' button.
+    Fill out the form that pops up. For the 'Git source' field,
+    use the SSH clone URL provided by your Git host. It shoud look
+    like this::
+
+        git@hostingsite.com:yourusername/yourreponame.git
+
+    or like this::
+
+        ssh://git@hostingsite.com/yourusername/yourreponame.git
+
+-   To make sure Relate can access your course content, you will need
+    an SSH keypair. Below the 'SSH private key' box in the course creation
+    form, there is a link to a tool (built into Relate) to help you create one.
+    Open that link in a new browser tab. Copy the 'private key' bit into the
+    'SSH private key' box on the course creation form. Next, find the
+    "Deployment key" section in the settings of your Git hosting site, and add
+    the public key there. On Github, this is under "Setting/Deploy keys". On
+    Gitlab, it is under "Settings/Repository/Deploy keys". For the title of the
+    key, you may choose any description you like.
+
+-   Fill out the rest of the form. You will want to pay special attention
+    to whether you want your course listed on the main page, whether
+    you would like it open for enrollment right away, who is allowed to enroll,
+    and whether the site is restricted to staff. Nearly all of these settings can be
+    changed later, under "Content/Edit Course", so if you make a mistake,
+    it's note the end of the world. The only things that have to be correct
+    at this point are the SSH settings, the the course identifier, course root,
+    and the settings for 'course root' and 'course file' (but, in all
+    likelihood, the defaults will be just fine).
+
+    .. note::
+
+        When choosing the 'course identifier', note that this will appear as
```

Comment:
```suggestion
        The course identifier is final and cannot be changed once the course
        is created.
        
        When choosing the course identifier, note that this will appear as
```